apple-pie/styles/style.css

```diff
@@ -1 +1,77 @@
+*{
+    margin: 0 auto;
+    padding: 0;
+}
 
+header {
+background-image: url(/apple-pie/images/apple-pie.jpg);
+position: relative;
+background-repeat: no-repeat;
+background-position: center;
+background-size:100%;
```

Comment:
Your background image is to tall, to solve this you can just add one line here in the `header` selector:
`height: 600px;`